pkg/proxy/ipvs/proxier.go

```diff
@@ -74,9 +74,6 @@ const (
 	// KubeLoadBalancerChain is the kubernetes chain for loadbalancer type service
 	KubeLoadBalancerChain utiliptables.Chain = "KUBE-LOAD-BALANCER"
 
-	// DefaultScheduler is the default ipvs scheduler algorithm - round robin.
-	DefaultScheduler = "rr"
```

Comment:
revisiting here, i'm not sure why the constant is deleted?
it is used in more than one location in .go files.

Reply:
The ipvs scheduler can be set by "---ipvs-scheduler string".  Usually, these default values are set in "pkg/proxy/apis/config/v1alpha1/defaults.go" instead of elsewhere.